packages/ERTP/src/issuer.chainmail

```diff
@@ -0,0 +1,213 @@
+/**
+ * The issuer cannot mint new amount, but it can create empty purses and
+ * payments. The issuer can also transform payments (splitting payments,
+ * combining payments, burning payments, and claiming payments
+ * exclusively). The issuer should be gotten from a trusted source and
+ * then relied upon as the decider of whether an untrusted payment is valid. 
+ */
+interface Issuer (Amount (Extent)) {
+  /**
+   * Get the Brand for this Issuer. The Brand indicates the kind of
+   * digital asset and is shared by the mint, the issuer, and any purses
+   * and payments of this particular kind. The brand is not closely
+   * held, so this function should not be trusted to identify an issuer
+   * alone. Fake digital assets and amount can use another issuer's brand.
+   */
+  getBrand() -> (Brand);
+
+  /* Get the allegedName for this mint/issuer */
+  allegedName() -> (allegedName);
+
+  /* Get the AmountMath for this Issuer. */
+  getAmountMath() -> (AmountMath);
+
+  /* Get the name of the MathHelpers for this Issuer. */
+  getMathHelpersName() -> (String);
+
+  /** Make an empty purse of this brand. */
+  makeEmptyPurse(memo :String) -> (Purse);
+
+  /**
+   * Get payment balance. Because the payment is not trusted, we
+   * cannot call a method on it directly, and must use the issuer
```

Comment:
> we cannot call a method on it directly

This comment is only relevant for people who have known ERTP before this PR, so i'm not sure it's relevant in this chainmail file

Reply:
Hmm, good point. I think we should keep it because I think it's helpful to understand why we are violating best practices in OO programming by not putting the functions about an object on the object.

Reply:
ok, sounds good

As a sidenote: after writing my comment, i kept thinking about it. I feel like one thing that might be hard to get is the history of design decisions. This history doesn't really belong neither in comments in the code, nor in the guide/reference documentation (i'm not sure about the chainmail files since they describe the API at a particular instant)
This history can be recovered from issues/PRs (and that's in part why i've sent my API feedback in the form of issues), but that's potentially a lot of archeology

Reply:
The comments in code, and especially in chainmail files, should be said to an audience that reads it in ignorance of the history. Generally that means they should be independent of the history. Sometimes though, even for an audience previously unaware of the history, a bit of the history of a concept helps explain the concept.

This one, I agree, should be independent of the history.

Reply:
However, I have another concern. `thing.getFoo()` is generally an instance method. It is an imperative of the receiver, i.e., the left operand. For occurrences of this style, perhaps we should consistently use `getFooOf` as in  `metaThing.getFooOf(thing)` ?

Reply:
I like `getFooOf` as the pattern!

Reply:
I switched from `issuer.getBalance(payment)` to `issuer.getAmountOf(payment)`